images/gitness/main.tf

```diff
@@ -0,0 +1,33 @@
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest-config" { source = "./config" }
+
+module "latest" {
+  source            = "../../tflib/publisher"
+  name              = basename(path.module)
+  target_repository = var.target_repository
+  config            = module.latest-config.config
+  build-dev         = true
+}
+
+module "test-latest" {
+  source = "./tests"
+  digest = module.latest.image_ref
+}
+
+module "version-tags" {
+  source  = "../../tflib/version-tags"
+  package = "gitness"
+  config  = module.latest.config
+}
+
+module "tagger" {
```

Comment:
This can just be an oci_tag for latest and latest-dev, since we won't version tag anything anymore.